config/samples/pods_with_pkcs11-luna.yaml

```diff
@@ -19,31 +19,44 @@ spec:
     database: BarbicanDatabasePassword
     service: BarbicanPassword
     simplecryptokek: BarbicanSimpleCryptoKEK
+    p11cryptologin: BarbicanP11Login
   preserveJobs: true
   customServiceConfig: |
     [DEFAULT]
     debug = True
+
+    [secretstore:pkcs11]
+    secret_store_plugin = store_crypto
+    crypto_plugin = p11_crypto
+
+    [p11_crypto_plugin]
+    plugin_name = PKCS11
+    library_path = /usr/local/luna/libs/64/libCryptoki2.so
+    token_serial_number = 12345
+    token_labels = some_token_label
+    slot_id = 123
```

Comment:
Only one of these values is necessary.  There is an order of priority and only one of the values will be used.  It may be better for the sample to only show one.

Reply:
Done